frontend/src/components/project/Buttons/ContactCreatorButton.js

```diff
@@ -8,12 +8,11 @@ import theme from "../../../themes/theme";
 import UserContext from "../../context/UserContext";
 
 const useStyles = makeStyles({
-  root: {
+  root: (props) => ({
     position: "relative",
-    height: 40,
-    width: 200,
+    width: props.customWidth ? props.customWidth : "auto",
```

Comment:
Please, check whether this property is really needed in all these places (I would expect we needed on root and that's it). Additionally, I would maybe prefer having 200px as default (like it was before) to have under control how it behaves on non-standard screens.

Reply:
I added a default width but I could not get rid of the width property in the child classes. The only way (I found) to make sure the button and card have the same width and the elements around the button know its width was to define the width within all three classes. But simply setting it to 100% in the child classes seems to be enough so I did that.